rrfs-test/ush/plot_jdiag.all.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+import os
+import sys
+import cartopy.crs as ccrs
+from cartopy.mpl.ticker import (LongitudeFormatter, LatitudeFormatter,
+                                LatitudeLocator, LongitudeLocator)
+from collections.abc import Iterable
+import numpy as np
+import matplotlib
+matplotlib.use('AGG')
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import matplotlib.axes as maxes
+import h5py as h5
+
+file = './jdiag_t133.nc4'			# = './obsout_da_amsua_n18.h5' 
+print('read file=', file)
+ncDB = h5.File(file,'r')
+obs = ncDB['/ObsValue/airTemperature']		# = ncDB['/ObsValue/brightnessTemperature'][:,4]     # for ch 5 
+omb = np.array(ncDB['/ombg/airTemperature'][:])	# = np.array(ncDB['/ombg/brightnessTemperature'[:,4])
+oma = np.array(ncDB['/oman/airTemperature'][:])	# = np.array(ncDB['/oman/brightnessTemperature'[:,4])
+qc  = ncDB['EffectiveQC2/airTemperature']		# = ncDB['/EffectiveQC2/brightnessTemperature'][:,4]
+hgt = ncDB['MetaData/height']
+lat = ncDB['MetaData/latitude']
+lon = ncDB['MetaData/longitude']
+omb[qc[:] !=0] = np.NaN
+oma[qc[:] !=0] = np.NaN
```

Comment:
This would be something more like:
```
fill_value = ds_ombg[obs_var].attrs.get('_FillValue', np.nan)
valid_mask = (ombg != fill_value) & (~np.isnan(obserr))  # Only use if EffectiveError0 is valid
ombg = ombg[valid_mask]  # Keep only assimilated observations
```